types/react-native/index.d.ts

```diff
@@ -388,7 +388,7 @@ export interface NativeTouchEvent {
     /**
      * The node id of the element receiving the touch event
      */
-    target: string;
+    target: NodeHandle;
```

Comment:
The documentation (JSDOC) clearly states this is a "node id". The official docs even say it's a `number` not a `string`: https://reactnative.dev/docs/pressevent#target

Reply:
So @eps1lon , what do you want to change here ?
As you can see, `type NodeHandle = number`.

Reply:
I didn't see that in this diff. Thanks for clarifying.

Reply:
You are welcome :)